checker/src/main/java/org/checkerframework/checker/index/inequality/LessThanTransfer.java

```diff
@@ -131,4 +140,17 @@ protected void refineGTE(
             return Collections.emptyList();
         }
     }
+
+    /**
+     * Return true if {@code receiver} is a double or float literal, which can't be parsed by {@link
+     * FlowExpressionParseUtil}.
+     */
+    private boolean isDoubleOrFloatLiteral(Receiver receiver) {
```

Comment:
Contrary to its documentation and name, this returns *false* if the argument is a double or float literal.  I think the logic needs to be changed here, and all uses need to be changed to `if (!isDoubleOrFloatLiteral(...))`.